jsonschema-validation.xml

```diff
@@ -575,21 +559,23 @@
                     validates against the provided schema.
                     Note the property name that the schema is testing will always be a string.
                 </t>
+                <t>
+                    A missing keyword has the same behavior as an empty schema.
```

Comment:
Should we specify this as a `true` schema instead of an empty schema?  Admittedly, the behavior is the same but `true` conveys the intent of permitting everything more clearly.